src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -760,6 +766,7 @@ def process_checkin(item: CheckinItem):
         ) as txn:
             _process_checkin(item, txn)
     except Exception:
+        metrics.incr("")
```

Comment:
was this intentional?

Reply:
Haha, no, I didn't realise that was still there